x-pack/test/api_integration/deployment_agnostic/apis/observability/ai_assistant/knowledge_base/helpers.ts

```diff
@@ -10,6 +10,8 @@ import { Client } from '@elastic/elasticsearch';
 import { AI_ASSISTANT_KB_INFERENCE_ID } from '@kbn/observability-ai-assistant-plugin/server/service/inference_endpoint';
 import { ToolingLog } from '@kbn/tooling-log';
 import { RetryService } from '@kbn/ftr-common-functional-services';
```

Comment:
Same as above. We should split this file up and move the methods into utility files here https://github.com/elastic/kibana/tree/main/x-pack/test/api_integration/deployment_agnostic/apis/observability/ai_assistant/utils

Reply:
... we could have a file called `utils/knowledge_base` for instance